src/App.js

```diff
@@ -6,6 +8,35 @@ import Hero from './components/Hero';
 import SocialMenu from './components/SocialMenu';
 
 function App() {
+  const [repos, setRepos] = useState([]);
+
+  useEffect(() => {
+    fetch('/api/repos')
+      .then(response => response.json())
+      .then(repos => {
+        Promise.all(
+          repos.map(repo =>
+            fetch('/api/commits', {
```

Comment:
Why don't you add fetching the commits to the lambda function? So that you can prepare the data and transform it in a way that the frontend needs it.